iogt/settings/production.py

```diff
@@ -1,7 +1,7 @@
 import os
 from .base import *
 
-DEBUG = True
+DEBUG = os.environ.get('DEBUG', 'False') == 'True'
```

Comment:
Should we use `bool(int(os.environ.get('DEBUG', 0))`

Reply:
Why?

In any case, I expect this to be a temporary option as DEBUG should never be 'True' in production.